lib/internal/source_map/source_map.js

```diff
@@ -112,14 +112,15 @@ class StringCharIterator {
  * @param {SourceMapV3} payload
  */
 class SourceMap {
+  #payload = null;
   #reverseMappingsBySourceURL = [];
   #mappings = [];
   #sources = {};
   #sourceContentByURL = {};
 
   /**
    * @constructor
-   * @param {SourceMapV3} payload
+   * @param {SourceMapV3} map
```

Comment:
You renamed the docs here but not the actual parameter :]

Reply:
I wiffled, and went back to `payload`, it's the terminology used more consistently in V8's SourceMap implementation.